### PR TITLE
Vatrp 3473

### PR DESCRIPTION
**Pull request by hrachyastepanyan**
VATRP- 3473: Shows list of contact when user clicks to chat icon

### CODE REVIEW

**Comment by zackm0571** [comment]
+1

File: linphone.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7261,8 +7293,8 @@
 				CLANG_ENABLE_OBJC_ARC = YES;
 				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "iPhone Developer: Gagik Martirosyan (9546538G66)";
```
Comment (kennethdick) [inline linphone.xcodeproj/project.pbxproj:7296]: Please revert

File: linphone.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -7308,8 +7340,8 @@
...
```
Comment (kennethdick) [inline linphone.xcodeproj/project.pbxproj:7343]: Revert

File: linphone.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -7214,8 +7246,8 @@
...
```
Comment (kennethdick) [inline linphone.xcodeproj/project.pbxproj:7249]: revert

File: linphone.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -7161,8 +7193,8 @@
...
```
Comment (kennethdick) [inline linphone.xcodeproj/project.pbxproj:7196]: revert

File: linphone.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -7118,8 +7150,8 @@
 				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
 				CLANG_WARN_EMPTY_BODY = YES;
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "iPhone Developer: Gagik Martirosyan (9546538G66)";
```
Comment (kennethdick) [inline linphone.xcodeproj/project.pbxproj:7153]: revert

File: linphone.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -7072,8 +7104,8 @@
...
```
Comment (kennethdick) [inline linphone.xcodeproj/project.pbxproj:7107]: revert

File: linphone.xcodeproj/project.pbxproj [hunk 7]
```diff
@@ -4912,7 +4941,7 @@
 				LastUpgradeCheck = 0700;
 				TargetAttributes = {
 					1D6058900D05DD3D006BFB54 = {
-						DevelopmentTeam = N35DF75MHS;
+						DevelopmentTeam = SJGPQWR594;
```
Comment (kennethdick) [inline linphone.xcodeproj/project.pbxproj:4944]: Revert

**Pull request merged**